todolist/app/components/Checkbox/layout.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { View, Text, StyleSheet, TouchableOpacity, CheckBox } from 'react-native';
+
+const styles = StyleSheet.create({
+  container: {
+    flexDirection: 'row',
+    justifyContent: 'space-between'
+  },
+  rightContainer: {
+    alignItems: 'center',
+    flexDirection: 'row',
+    justifyContent: 'center'
+  },
+  item: {
+    color: 'black',
+    height: 44,
+    padding: 10
+  },
+  removeButton: {
+    padding: 10
```

Comment:
extract 10 to a const outside the stylesheet object to avoid repeting. i.e. you can use PADDING_SIZE = 10, so you can use that const twice in this file. Just do this if a value repeats itself